### ISSUE TITLE
add require-v-for-key options for simple DOM content

### ISSUE THREAD

**Issue by plehnen** [vuejs/eslint-plugin-vue#851]
**Please describe what the rule should do:**

vue/require-v-for-key should have an optional exception for primitive HTML code.

Quoting from the [official docs](https://vuejs.org/v2/guide/list.html#key):
> It is recommended to provide a key with v-for whenever possible, **unless the iterated DOM content is simple**, or you are intentionally relying on the default behavior for performance gains.
(Emver

**What category should the rule belong to?**

- [ ] Enforces code style
- [ ] Warns about a potential error
- [X] Suggests an alternate way of doing something
- [ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**
Imaging you have this list:
```vue
<dl class="list-desc-grid">
	<template v-for="(desc, index) in order.descriptions">
		<dt :key="'dt' + index">{{ desc.name }}:</dt>
		<dd :key="'dd' + index">{{ desc.value }}</dd>
	</template>
</dl>
```
The name could be identical in several rows, as well as the value.
It looks very ugly to write those :keys and therefore it would be nice to have the possibility to rule out such simple DOM contents.

```js
'vue/require-v-for-key': [
	'error',
	{
		ignore: ['dt', 'dd']
	},
],
```

Or maybe to only make this an "error" with real components?

**Comment by ota-meshi** [comment]
If you don't want the key required, you need to turn off the `vue/require-v-for-key` rule in your configuration file.
